GVRf/Framework/framework/src/main/jni/objects/textures/render_texture.cpp

```diff
@@ -23,215 +23,438 @@
 #include "eglextension/msaa/msaa.h"
 
 namespace gvr {
-RenderTexture::RenderTexture(int width, int height, GLTexture* tex) :
+GLFrameBuffer *RenderTexture::renderTexture_gl_read_buffer = nullptr;
```

Comment:
I clearly misunderstand something. You will explain tomorrow in person. Last question, isn't the renderTexture_gl_read_buffer used only for the multiview case?